packages/lodestar/src/api/rest/routes/validator/duties.ts

```diff
@@ -0,0 +1,53 @@
+import {IFastifyServer} from "../../index";
+import fastify, {DefaultQuery} from "fastify";
+import {IApiModules} from "../../../interface";
+import {getValidatorDuties} from "../../../impl/validator";
+import {IncomingMessage, Server, ServerResponse} from "http";
+import {toRestJson} from "../../utils";
+
+interface Query extends DefaultQuery {
+  validator_pubkeys: string[];
+  epoch: number;
+}
+
+
+const opts: fastify.RouteShorthandOptions<Server, IncomingMessage, ServerResponse, Query> = {
+  schema: {
+    querystring: {
+      type: 'object',
+      required: ["validator_pubkeys", "epoch"],
+      properties: {
+        "validator_pubkeys": {
+          type: 'array',
+          maxItems: 5,
+          items: {
+            types: "string"
+          }
+        },
+        epoch: {
+          type: "integer",
+          minimum: 0
+        }
+      }
+    },
+  }
+};
+
+export const registerDutiesEndpoint = (fastify: IFastifyServer, modules: IApiModules): void => {
+  fastify.get<Query>(
+    '/duties',
```

Comment:
probably `/validator/duties` according to https://github.com/ethereum/eth2.0-specs/blob/dev/specs/validator/beacon_node_oapi.yaml? Same for other end points.

Reply:
https://github.com/ChainSafe/lodestar/blob/920161e12029de8aa786cd82b324d90b63d5ca8d/packages/lodestar/src/api/rest/index.ts#L50